controller_manager_ns: controller_manager
controller_list:
  - name: arm/arm_joint_controller
    action_ns: follow_joint_trajectory
    type: FollowJointTrajectory
    default: true
    joints:
      - joint1
      - joint2
      - joint3
      - joint4
      - joint5
      - joint6

    gains:
      joint1: { p: 1000.0, i: 0.0, d: 0.1, i_clamp: 0.0 }
      joint2: { p: 1000.0, i: 0.0, d: 0.1, i_clamp: 0.0 }
      joint3: { p: 1000.0, i: 0.0, d: 0.1, i_clamp: 0.0 }
      joint4: { p: 1000.0, i: 0.0, d: 0.1, i_clamp: 0.0 }
      joint5: { p: 1000.0, i: 0.0, d: 0.1, i_clamp: 0.0 }
      joint6: { p: 1000.0, i: 0.0, d: 0.1, i_clamp: 0.0 }

  - name: arm/gripper_controller
    action_ns: follow_joint_trajectory
    type: FollowJointTrajectory
    default: true
    joints:
      - finger_joint1
      - finger_joint2
    gains:
      finger_joint1: { p: 50.0, d: 1.0, i: 0.01, i_clamp: 1.0 }
      finger_joint2: { p: 50.0, d: 1.0, i: 0.01, i_clamp: 1.0 }
